trace/google/cloud/trace/trace_span.py

```diff
@@ -105,11 +105,11 @@ def span(self, name='child_span'):
         self._child_spans.append(child_span)
         return child_span
 
-    def set_start_time(self):
+    def start(self):
```

Comment:
Is this public API needed? Usually when you create a Span object you also start it.